src/time_zone_lookup.cc

```diff
@@ -37,7 +37,12 @@
 #include <sdkddkver.h>
 // Include only when the SDK is for Windows 10 (and later), and the binary is
 // targeted for Windows XP and later.
-#if defined(_WIN32_WINNT_WIN10) && (_WIN32_WINNT >= _WIN32_WINNT_WINXP)
+// Note: Windows SDK added windows.globalization.h file in 10.0.10240.0,
+//   but mingw added it in 10.0.22621.0, so the version is checked against
+//   10.0.22621.0 in mingw.
+#if ((defined(_WIN32_WINNT_WIN10) && !defined(__MINGW32__)) ||        \
+     (defined(NTDDI_WIN10_NI) && NTDDI_VERSION >= NTDDI_WIN10_NI)) && \
+    (_WIN32_WINNT >= _WIN32_WINNT_WINXP)
```

Comment:
Should this repeated expression be encapsulated in a single, say, "USE_WIN32_LOCAL_TIME_ZONE"?

Reply:
Done.